tools/functional-tester/etcd-tester/tester.go

```diff
@@ -115,3 +121,77 @@ func (s *Status) setCase(c int) {
 	defer s.mu.Unlock()
 	s.Case = c
 }
+
+// checkConsistency stops the cluster for a moment and get the hashes of KV storages.
+// TODO: instead of logging to stdout, record it as an event to database.
+func (c *cluster) checkConsistency() error {
+	log.Println("canceling all stressers")
```

Comment:
let's print out the purpose of the action rather than the action itself. For example, starting consistency checking...